src/Schema/Column/AbstractColumnFactory.php

```diff
@@ -210,4 +227,57 @@ protected function isType(string $type): bool
             default => false,
         };
     }
+
+    /**
+     * Converts column's default value according to {@see ColumnSchemaInterface::getPhpType()} after retrieval from the
+     * database.
+     *
+     * @param string|null $defaultValue The default value retrieved from the database.
+     * @param ColumnSchemaInterface $column The column schema object.
+     *
+     * @return mixed The normalized default value.
+     */
+    protected function normalizeDefaultValue(string|null $defaultValue, ColumnSchemaInterface $column): mixed
+    {
+        if (
+            $defaultValue === null
+            || $defaultValue === ''
```

Comment:
Are you sure that empty string should cast to null? Empty string can be default value.